main.go

```diff
@@ -68,15 +60,15 @@ func initAndConnectEtherium() (*ethclient.Client, error) {
 	return client, nil
 }
 
-func getLMCAddressesByTenant(collection *mongo.Collection, tenant string) []*models.LMCAddress {
+func getLMCAddressesByTenant(collection *mongo.Collection, tenant string, context context.Context) []*models.LMCAddress {
```

Comment:
The non-written convention in Go is that the Context (if it exists) is the first argument of a function. It is not wrong like this, but it is not really idiomatic Go. You can see that also in the function calls for the MongoDB.

Reply:
Done